### PR TITLE
Notes 1-26

### PR DESCRIPTION
**Pull request by 03222000ls**

### CODE REVIEW

File: notes/2022-01-26.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    - Are useful for working with less data to create strong inferences
+        - Working with things when not a lot of data is available
+    - Assumptions, expressed in a probability distribution
+- Posterior
+    - Inference given regularizer: Likelihood...
+    - Most common posterior probability distribution we're doing: Probability of parameters given data
+- Point Estimate
+    - This are the single values produced after training (weights)
+    - Posterior mean
+![very separable data](/images/00-eytrmV6b6ZOCwq9UpSYeI9oiiuA3-7daeae0f-b595-4ce3-bdd1-dee4e59d1ccd.svg)
```
Comment (brownsarahm) [inline notes/2022-01-26.md:49]: the part in the ` []` should be alt text (a description of the image)

**Comment by brownsarahm** [comment]
Can we get each `.svg` file renamed to something descriptive?  The easiest place to do that might be from [the branch the pr is from](https://github.com/03222000ls/ml4scisoc.github.io/tree/patch-1/images) where you can view them as images. Then you can "Edit" on github to change the file name.

File: notes/2022-01-26.md (hunk 1 above)
Comment (03222000ls) [reply notes/2022-01-26.md]: I've updated the description of the images, but I'm not sure how in depth I was supposed to be. If I should be more or less descriptive let me know :)

Comment (brownsarahm) [reply notes/2022-01-26.md]: that's good enought

File: notes/2022-01-26.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    - (e.g.) Murder mystery --> murderer variable can be Grey or Auburn
+![example marginal distribution from murder mystery text](/images/marginalDistribution.svg)
+
+- Marginal probability
+    - (Section 1.2 -- A Model of Murder)
+    - "Probability of one event in the presence of all (or subset) outcomes of the other random variable..." (https://machinelearningmastery.com/joint-marginal-and-conditional-probability-for-machine-learning/)
+- Maximum Likelihood Estimation
+    - Assume a distribution, our goal will be to find the theta (parameter)
+    - Maximizing, find parameters that will give us the highest probability (finding the one--parameter--that fits best)
+- elicitation - study of how to elicit...
```
Comment (brownsarahm) [inline notes/2022-01-26.md:64]:
```suggestion
- elicitation - an interdisciplinary field in statistics and psychology; study of how to get an expert's distribution for how likely an event is to occur. 
```

**Comment by brownsarahm** [comment]
We still need the rest of the images added to the notes.

**Comment by brownsarahm** [comment]
I'm going to merge this so that most of it is visible.

**Pull request merged**